CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.5.5 (2021-07-12)
+
+- Fixed the modal component to properly skip disabled elements with a `tabIndex` attribute when focusing the first focusable child when the modal is created. [#158](https://github.com/blackbaud/skyux-modals/pull/158)
```

Comment:
```suggestion
- Fixed the modal component so that when the modal is created, it properly skips disabled elements with a `tabIndex` attribute when focusing the first focusable child. [#158](https://github.com/blackbaud/skyux-modals/pull/158)
```